README.md

```diff
@@ -4,6 +4,7 @@ community-scripts
 A collection of ZAP scripts provided by the community, i.e. you lot :)
 
 The easiest way to use this repo in ZAP is to install the 'Community Scripts' add-on from the ZAP Marketplace.
+Then, enabling [`Scripts Tab`](https://www.zaproxy.org/docs/desktop/addons/script-console/tree/) by `View->Show Tab->Scripts Tab` to manage the scripts.
```

Comment:
The help page is already being linked below.

Reply:
The team talked about this a bit today. Here's our suggestion:

1. Edit the text to be:
> Display the [`Scripts Tab`](https://www.zaproxy.org/docs/desktop/addons/script-console/tree/) by `View->Show Tab->Scripts Tab` to manage the scripts.
2. Move it down below `For more information on ZAP scripts see:`, making it the first bullet and change that lead-in text to be `For more information on ZAP scripts:` (dropping the see).

Let me know if anything there isn't clear.